### PR TITLE
r.maxent.train.py: fixes wrong handling of end of line character

### PR DESCRIPTION
**Pull request by ecodiv**
This patch ensures a better cross-platform handling of end of line characters when creating a list of addons using g.extension.

### CODE REVIEW

File: src/raster/r.maxent.train/r.maxent.train.py [hunk 1]
```diff
@@ -810,10 +810,11 @@ def main(options, flags):
     # Get list with all files in the output folder
     # -----------------------------------------------------------------
     all_files = all_files = os.listdir(options["outputdirectory"])
-    # Check if v.db.pyupdate is installed
-    plugins_installed = gs.read_command(
+    # Create list of addons. Is later used to check if v.db.pyupdate is installed
+    outputs = gs.read_command(
         "g.extension", flags="a", quiet=function_verbosity
-    ).split("\n")
+    )
```
[the remark this answers is not in the document]
Comment (ecodiv) [reply src/raster/r.maxent.train/r.maxent.train.py]: @wenzeslaus do you know how to deal with this? The proposed change is in the old code, so even if I wanted to, I cannot change that. I tried to commit the suggestion, resulting in the message that it is not possible to change already deleted code (which makes sense, I would say).

[the remark this answers is not in the document]
Comment (echoix) [reply src/raster/r.maxent.train/r.maxent.train.py]: We can "resolve" the comment suggestion, or delete it if black was successful

[the remark this answers is not in the document]
Comment (ecodiv) [reply src/raster/r.maxent.train/r.maxent.train.py]: > We can "resolve" the comment suggestion, or delete it if black was successful

Thanks. I ran the black test again and it failed again. I don't understand the error message though. https://github.com/OSGeo/grass-addons/actions/runs/12315208744/job/34401054587#step:6:1

[the remark this answers is not in the document]
Comment (echoix) [reply src/raster/r.maxent.train/r.maxent.train.py]: > > We can "resolve" the comment suggestion, or delete it if black was successful
> 
> Thanks. I ran the black test again and it failed again. I don't understand the error message though. https://github.com/OSGeo/grass-addons/actions/runs/12315208744/job/34401054587#step:6:1

I wouldn't expect rerunning that exact CI run to change something if the best-effort suggestions don't work (due to limitations of what is possible to write review suggestions on). Just run `black .` locally, or `pre-commit run -a` locally, to make the real fixes.

**Pull request merged**